utils/RunONNXModel.py

```diff
@@ -129,6 +138,13 @@ def valid_onnx_input(fname):
                     " uint64, int64, float16, float32, float64")
 
 args = parser.parse_args()
+if (args.verify and not (args.verify_with_softmax or args.verify_every_value)):
+    raise RuntimeError("Choose verification mode: --verify-with-softmax or "
+                       "--verify-every-value or both")
+if (args.verify_with_softmax and (not args.verify)):
+    raise RuntimeError("Must specify --verify to use --verify-with-softmax")
+if (args.verify_every_value and (not args.verify)):
+    raise RuntimeError("Must specify --verify to use --verify-every-value")
```

Comment:
> LGTM, please add before merging in the help output which options are default? By using the [ ] in the top level usage, it does not seems to imply one of them is mandatory.

I do the check here since I haven't found ways to make dependent arguments, say, once `--verify` is given, one must have `--verify-with-softmax` or `--verify-every-value` or both.